tools/generate_dalite_passport.py

```diff
@@ -0,0 +1,26 @@
+"""Utility that allows to generate passports encoded for this xblock."""
+import argparse
+from dalite_xblock.passport_utils import DaliteLtiPassport, prepare_passport
+
+
+def main():
+    """Entrypoint for this script."""
+    parser = argparse.ArgumentParser(description='Create dalite passport')
+    parser.add_argument('--passport-name', help='Name of this passport', required=True)
```

Comment:
https://github.com/open-craft/xblock-dalite/pull/2/commits/2f0039a07a0efca9baf21dafa4af06b9cd554e76#diff-6ac57e7e8afa8bd1d6836a7f77908adeR20 . 

BTW: Passport id is slightly better as it matches the wording in studio.